cmd/keeper/keeper.go

```diff
@@ -278,14 +278,20 @@ func (p *PostgresKeeper) createPGParameters(db *cluster.DB) common.Parameters {
 		for _, synchronousStandby := range db.Spec.SynchronousStandbys {
 			synchronousStandbys = append(synchronousStandbys, common.StolonName(synchronousStandby))
 		}
+
+		var syncStandbyNumber int = len(synchronousStandbys)
+		if len(db.Spec.AdditionalSyncStandbyNames) > 0 {
+			synchronousStandbys = append(synchronousStandbys, db.Spec.AdditionalSyncStandbyNames)
+		}
+
 		// TODO(sgotti) Find a way to detect the pg major version also
 		// when the instance is empty. Parsing the postgres --version
 		// is not always reliable (for example we can get `10devel` for
 		// development version).
 		// Now this will lead to problems starting the instance (or
 		// warning reloading the parameters) with postgresql < 9.6
-		if len(synchronousStandbys) > 1 {
-			parameters["synchronous_standby_names"] = fmt.Sprintf("%d (%s)", len(synchronousStandbys), strings.Join(synchronousStandbys, ","))
+		if syncStandbyNumber > 1 {
```

Comment:
Probably here there's some logic that needs to be detailed in a comment. If I got it right you want to define the additionalsyncstandbynames only if there's at least one defined synchronous standby?

I won't do all of this here. This is something that should be handled by the sentinel. So, inside the sentinel I'll just add the additional synchronous standby in db.Spec.SynchronousStandby.